PA1Q3

```diff
@@ -0,0 +1,34 @@
+.data
+#Data below is stored in memmory
+fixedV: .word 1
+currentV: .word 0
+newV:.word 1
+prompt1: .asciiz "\nAnswer\n" 
+.text
+
+loop: 
+
+lw $t0,fixedV # this value never changes. It is always 1
```

Comment:
-2 Lines 11, 13-15:  You're loading (from memory) the value 1 into $t0 each iteration, and setting $t2 to 2, then you're multiplying $t0 and $t2 to get the value 2 in $t3, then adding $t3 to current value in $t4.....why?  Very inefficient to do all of this, just use 'addi $t4, $t4, 2' and be done with it.